app/views/layouts/application.html.erb

```diff
@@ -36,8 +36,10 @@
               </li>
               <li><%= link_to_classroom %></li>
               <li><%= link_to_bibliotheca %></li>
-              <li><%= solve_discussions_link %></li>
-              <li><%= user_discussions_link %></li>
+              <% unless current_user.currently_in_exam? %>
```

Comment:
@julian-berbel See that both user_discussions_link and solve_discussions_link uses a method called discussions_link. I think it's cleaner to check the condition in that helper method, as we are already checking for organization's flag